datastream-server-restli/src/test/java/com/linkedin/datastream/server/TestCoordinator.java

```diff
@@ -656,6 +657,86 @@ public void testCoordinationWithStickyMulticastStrategy() throws Exception {
     zkClient.close();
   }
 
+  /**
+   * testCoordinationWithStickyMulticastStrategyAndMaxTaskLimit is a test to verify that assignment cannot complete
+   * when the tasks per instance is exceeded by the configured threshold
+   * <ul>
+   *     <li>Create a coordinator with CoordinatorConfig.CONFIG_MAX_DATASTREAM_TASKS_PER_INSTANCE set to 5</li>
+   *     <li>create 3 instances and 4 datastreams, verify assignment cannot complete</li>
+   *     <li>create another instance, verify assignment completes now</li>
+   * </ul>
+   */
+  @Test
+  public void testCoordinationWithStickyMulticastStrategyAndMaxTaskLimit() throws Exception {
+    String testCluster = "testCoordinationMaxTasksPerInstance";
+    String testConnectorType = "testConnectorType";
+    Properties properties = new Properties();
+    // Set max tasks per instance to 5, since some instances will get 6 tasks if 4 datastreams with 4 tasks each are
+    // created across 3 instances. With 4 instances, the tasks per instance will be less than 5.
+    properties.put(CoordinatorConfig.CONFIG_MAX_DATASTREAM_TASKS_PER_INSTANCE, "5");
+    Coordinator instance1 = createCoordinator(_zkConnectionString, testCluster, properties);
+
+    TestHookConnector connector1 = new TestHookConnector("connector1", testConnectorType);
+    //Question why the multicast strategy is within one coordinator rather than shared between list of coordinators
```

Comment:
Is the comment a todo that you had to remove?

Reply:
done - removed it. was a copy paste error